dbt/tests/generic/test_expression_is_true.sql

```diff
@@ -1,13 +1,17 @@
 -- Filter for rows where a given `expression` is false.
 --
 -- Adapted from `dbt_utils.expression_is_true`, and extended to support
--- an optional `select_columns` option representing an array of columns to
--- select for failing rows in addition to `column_name`. If no `select_columns`
--- array is provided, defaults to selecting the column represented by
--- `column_name`; if `column_name` is also missing, falls back to selecting 1
--- for failing rows.
-{% test expression_is_true(model, column_name, expression, select_columns=[]) %}
-    {%- set select_columns_csv = select_columns | join(", ") -%}
+-- an optional `additional_select_columns` option representing an array of
+-- columns to select for failing rows in addition to `column_name`. If no
+-- `additional_select_columns` array is provided, defaults to selecting the
+-- column represented by `column_name`; if `column_name` is also missing, falls
+-- back to selecting 1 for failing rows.
+{% test expression_is_true(
+    model, column_name, expression, additional_select_columns=[]
```

Comment:
Previously, we had named this attribute `select_columns` rather than `additional_select_columns` because it was rare for `expression_is_true` to be defined on a column as opposed to on a model; when defined on a model, `additional_select_columns` might be a confusing name because no other columns are being selected by default.  (When defined on a column, however, the column corresponding to `column_name` is selected by default, so the parameter name is clear.)

However, I think standardization of the generic interfaces is more important than maximum clarity for `expression_is_true` in particular; there is also no guarantee that the balance of model vs. column tests won't change for `expression_is_true` in the future. As such, I've gone ahead and refactored the parameter name here to align it with the rest of the generics.